kotlin/codegen/pom.xml

```diff
@@ -56,32 +56,31 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <version>1.0</version>
```

Comment:
versions should go in `<dependencyManagement>` section of root pom.

Reply:
Part of a4c55f7